arangod/Agency/Store.cpp

```diff
@@ -692,7 +692,41 @@ bool Store::applies(arangodb::velocypack::Slice const& transaction) {
     Slice value = transaction.get(key);
 
     if (value.isObject() && value.hasKey("op")) {
-      _node.hasAsWritableNode(abskeys.at(i)).first.applieOp(value);
+      if (value.get("op").isEqualString("delete") ||
+          value.get("op").isEqualString("replace") ||
+          value.get("op").isEqualString("erase")) {
+        if (!_node.has(abskeys.at(i))) {
+          continue;
+        }
+      } 
+      auto uri = abskeys.at(i);
```

Comment:
I think we need more `uri` normalization. For example `//` is transformed to `/`. Is this handled?

Reply:
i am not changing this for this PR.

Reply:
So then this is a change in behavior? Because previously the result of `Node::uri()` was used as `uri`, in this version we take the user provided string and only make sure that it is prefixed with `/`.

Reply:
It results to same.